api/diagnostics.go

```diff
@@ -303,7 +304,7 @@ func (j *DiagnosticsJob) collectDataFromNodes(ctx context.Context, nodes []dcos.
 	fetchStatusUpdate := make(chan fetcher.StatusUpdate)
 	fetchResponse := make(chan fetcher.BulkResponse)
 
-	numberOfWorkers := 1 //TODO(janisz): make number of workers configurable
+	numberOfWorkers := j.Cfg.FlagDiagnosticsBundleFetchersCount
```

Comment:
Would you be able to add a unit test that asserts that the number of fetchers created corresponds to the configuration flag?

Reply:
Hmm, I'm not sure how to test it. In theory number of workesrs is transparent we could try to catch concurrent requests but I don't think this is easy

Reply:
Okay, let's leave this as an exercise for a rainy day...